tests/phpunit/tests/blocks/updateIgnoredHookedBlocksPostMeta.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * Tests for update_ignored_hooked_blocks_postmeta
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ *
+ * @since 6.6.0
+ *
+ * @group blocks
+ */
+class Tests_Blocks_UpdateIgnoredHookedBlocksPostMeta extends WP_UnitTestCase {
+
+	/**
+	 * Original markup.
+	 *
+	 * @var string
+	 */
+	protected static $original_markup;
+
+	/**
+	 * Post object.
+	 *
+	 * @var object
+	 */
+	protected static $navigation_post;
+
+	/**
+	 * Setup method.
+	 */
+	public static function wpSetUpBeforeClass() {
+		//self::$original_markup = '<!-- wp:navigation-link {"label":"News & About","type":"page","id":2,"url":"http://localhost:8888/?page_id=2","kind":"post-type"} /-->';
+
+		self::$navigation_post = self::factory()->post->create_and_get(
+			array(
+				'post_type'    => 'wp_navigation',
+				'post_title'   => 'Navigation Menu',
+				'post_content' => 'Original content',
+			)
+		);
+	}
+
+	/**
+	 * Tear down each test method.
+	 */
+	public function tear_down() {
+		$registry = WP_Block_Type_Registry::get_instance();
+
+		if ( $registry->is_registered( 'tests/my-block' ) ) {
+			$registry->unregister( 'tests/my-block' );
+		}
+
+		parent::tear_down();
+	}
+
+	/**
+	 * @covers ::update_ignored_hooked_blocks_postmeta
+	 */
+	public function test_block_core_navigation_update_ignore_hooked_blocks_meta_preserves_entities() {
+		if ( ! function_exists( 'set_ignored_hooked_blocks_metadata' ) ) {
+			$this->markTestSkipped( 'Test skipped on WordPress versions that do not included required Block Hooks functionality.' );
+		}
```

Comment:
We don't have that excuse here -- we're in Core, where we know that the function does exist 😄 

```suggestion
		if ( ! function_exists( 'set_ignored_hooked_blocks_metadata' ) ) {
			$this->markTestSkipped( 'Test skipped on WordPress versions that do not included required Block Hooks functionality.' );
		}
```

Reply:
>We don't have that excuse here -- we're in Core, where we know that the function does exist 😄

Thanks, I wasn't sure whether these also run against previous WP versions for back-compat reasons.

Reply:
572ed7102de9ee0bb9c0af98975106219e1b24ad